bin/deliver_poste_messages

```diff
@@ -270,18 +270,26 @@ def main()
     until queue.empty?
       next unless delivery = queue.pop(true) rescue nil
 
+      sent_at = DateTime.now
+
       begin
         deliverer.send delivery
-      rescue Net::SMTPSyntaxError => e
-        puts "Unable to deliver #{delivery[:id]} to '#{delivery[:contact_email]}' because '#{e.message.to_s.strip}'"
-      rescue Net::SMTPFatalError => e
+      rescue Net::SMTPSyntaxError, Net::SMTPFatalError => e
         puts "Unable to deliver #{delivery[:id]} to '#{delivery[:contact_email]}' because '#{e.message.to_s.strip}'"
+      rescue RuntimeError => e
+        if e.message.to_s.start_with? 'ABORT:'
```

Comment:
why use the message string in a RuntimeError instead of defining a new error class?